src/Microsoft.VisualStudio.Editors/ResourceEditor/Resource.vb

```diff
@@ -1603,6 +1606,45 @@ Namespace Microsoft.VisualStudio.Editors.ResourceEditor
             End If
         End Function
 
+        ''' <remarks>
+        ''' This is a bit of a hack to allow the designer to work properly with netstandard2.0 projects.
+        ''' Our multi-targeting support quite correctly tells us that types like System.String are found
+        ''' in netstandard.dll. However, VS (and thus the resx designer) and resgen.exe run on the net462
+        ''' framework, which does not have a netstandard.dll. As such the designer doesn't understand the
+        ''' types involved and crashes while trying to create or update resource strings. Even if we
+        ''' could get past that, resgen.exe would see the references to netstandard.dll in the .resx
+        ''' file, fail to locate it, and die while trying to produce the .designer.cs file.
+        ''' 
+        ''' The workaround here is to check if the assembly-qualified type name refers to a type in 
+        ''' netstandard.dll, and if it does swap in mscorlib.dll instead. This allows the designer to
+        ''' work. The updated assembly-qualified typename also makes its way into the .resx file,
+        ''' allowing resgen.exe to work properly and produce the .designer.cs file.
+        ''' 
+        ''' This does not affect the runtime behavior of the app in any way as the types in the .resx
+        ''' file are only used to produce the .designer.cs file, which is still built against
+        ''' netstandard.dll.
+        ''' </remarks>
+        Private Shared Function AdjustAssemblyQualifiedNameForNetStandard(AssemblyQualifiedName As String) As String
+            Static MscorlibAssemblyName As AssemblyName
+
+            If AssemblyQualifiedName.Contains("netstandard") Then
```

Comment:
That's intentional--I want to avoid any allocations in the case that we don't need to update the assembly.

However, I realized that in cleaning up this code for review I deleted the part where I explicitly check that the assembly name is "netstandard", which is rather important. Let me fix that up...